src/lib/containers/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -471,11 +476,11 @@ const FacetNavPanel: React.FunctionComponent<FacetNavPanelProps> = ({
             {({ size }) => (
               <div className={getClassNameForPlotDiv(isExpanded, plotType)}>
                 <Plot
+                  key={`${facetToPlot.columnName}-${size.width}`}
                   layout={layout}
                   data={plotData?.data ?? []}
-                  style={getPlotStyle(size.width, plotType)}
-                  config={{ displayModeBar: false, responsive: true }}
```

Comment:
Did removing `responsive: true` have any particular effect?

Reply:
No, but I want to reduce any extra work that they might be doing within their component!